crates/holochain/src/core/ribosome/host_fn/block_agent.rs

```diff
@@ -91,6 +91,7 @@ mod test {
 
     #[tokio::test(flavor = "multi_thread")]
     #[cfg(feature = "slow_tests")]
+    #[ignore = "flaky"]
     async fn zome_call_get_block() {
```

Comment:
After running on CI, this one is flaky too.

Reply:
I think there's some commented out functionality under block/unlock. Should this one have a comment for us to take another look with integration? 

Equally, I don't think this is really usable until we can actually block people which I think we're losing with K2?

Reply:
Updated: https://github.com/holochain/holochain/pull/4838/commits/0f4281bbc6c3ca40885ddb4f2e0e2b2d6074e1bb